codespell_lib/data/dictionary.txt

```diff
@@ -11955,6 +11964,8 @@ prority->priority
 proseletyzing->proselytizing
 prosess->process
 prosessor->processor
+prosses->process, pocesses, possess, prosses,
```

Comment:
`prosses` is an [actual word](https://1word.ws/prosses) although rare. Left it in nevertheless.

Reply:
fixed that misspelling from `pocesses` to `processes`